Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPalette.cs

```diff
@@ -31,7 +31,13 @@ private class ImageReverseDictionary : Dictionary<string, Bitmap> { }
         #endregion
 
         #region Static Fields
-        private static readonly int _paletteVersion = 19;
+        //private static readonly int _paletteVersion = 19;
```

Comment:
Delete this as it is confusing the value below

Reply:
Removed field, left it in to see if anything else was using it